faiss.py

```diff
@@ -0,0 +1,35 @@
+import pickle
+import faiss
+
+class FAISS_index:
```

Comment:
For classes in Python we use CamelCaseNotation.
So name class as "FaissIndexer"
Also, you can make interface Indexer and then implement it in this class.